sklearn/metrics/_pairwise_distances_reduction/_dispatcher.py

```diff
@@ -489,8 +489,8 @@ def compute(
         Y,
         k,
         weights,
-        labels,
-        unique_labels,
+        class_membership,
+        unique_classes,
```

Comment:
If I had the chance to rename these, I think I'll go with:

- `class_membership` -> `y_labels`: Directly connects this parameter with `y`
- `unique_classes` -> `labels`: This would be consistent with the `labels` kwargs in the `metrics` module

Reply:
This indeed looks better (I trust you for coming up with the best, appropriate names).

Reply:
Small heads-up, @Micky774. Is there something missing for this PR? :)